### ISSUE TITLE
Breaking change in V2 API?

### ISSUE THREAD

**Issue by henrypinkard** [datadryad/dryad-product-roadmap#3543]
I've written code that uses the v2 API to automatically download files, and it used to find the correct URL of a file to download using the `'stash:file-download'` field. However, this recently stopped working and now it appears the same information can be found in ['stash:download']. Is this expected? I am unintentionally relying on a non-stable part of the API?

see https://github.com/Waller-Lab/BSCCM/issues/4

**Comment by ahamelers** [comment]
Hello Henry,

We recently found that both `stash:download` and `stash:file-download` were present, with the same exact content. Checking our documentation and its history, we found only `stash:download` had ever been documented (https://datadryad.org/stash/api#/versions/get_versions__id__files). Therefore we stopped maintaining the undocumented `file-download` as a public method. I apologize for the inconvenience! We are moving to a method of updating the API in which changes and deprecations will be much more clear and advance warning will be given. In this case, we were not aware anyone was using the undocumented method.

Best,
Audrey
